<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1505</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #f5f7fa, stop:1 #c3cfe2);
/* Light blue gradient */</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="statusComboBox">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>540</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    font-family: &quot;Rancho&quot;;
    font-size: 18px;
    color: black;
    background: white;
    padding: 5px;
    border: 1px solid #d0d0d0;
}</string>
    </property>
    <item>
     <property name="text">
      <string>Pending</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Draft</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Queued</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>590</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 26px;
font-family: &quot;Rancho&quot;;
color: red;</string>
    </property>
    <property name="text">
     <string>Review  :</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-60</x>
      <y>50</y>
      <width>1601</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
    background-color: black;
    min-height: 2px;  /* Line thickness */
    max-height: 2px;
    border: none;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>540</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 26px;
font-family: &quot;Rancho&quot;;
color: red;</string>
    </property>
    <property name="text">
     <string>Status :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ReferenceTextEdit">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>470</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">   font-size:18px;
    background-color: white;  
    color: black;            
    border: 1px solid #CCCCCC;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="reviewTextEdit">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>590</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">   font-size:18px;
    background-color: white;  
    color: black;            
    border: 1px solid #CCCCCC;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>691</width>
      <height>751</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: lightyellow;
    
</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 26px;
font-family: &quot;Rancho&quot;;
color: blue;
font-weight: bold; </string>
     </property>
     <property name="text">
      <string>INCOME</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 26px;
font-family: &quot;Rancho&quot;;
color: blue;
font-weight: bold; 
</string>
     </property>
     <property name="text">
      <string>EXPENSES</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 22px;
font-family: &quot;Rancho&quot;;
color: black
</string>
     </property>
     <property name="text">
      <string>Sources :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>160</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 22px;
font-family: &quot;Rancho&quot;;
color: black;</string>
     </property>
     <property name="text">
      <string> Amount :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 22px;
font-family: &quot;Rancho&quot;;
color: black;
</string>
     </property>
     <property name="text">
      <string>Currency  :</string>
     </property>
    </widget>
    <widget class="QComboBox" name="incomeSouceComboBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
    font-family: &quot;Rancho&quot;;
    font-size: 16px;
    color: black;
    background: white;
    padding: 5px;
    border: 1px solid #d0d0d0;
}
</string>
     </property>
     <item>
      <property name="text">
       <string>Bonus</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Investments</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Other</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Salary</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>90</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 22px;
font-family: &quot;Rancho&quot;;
color: black;
</string>
     </property>
     <property name="text">
      <string>Category    :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>150</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 22px;
font-family: &quot;Rancho&quot;;
color: black;</string>
     </property>
     <property name="text">
      <string>Amount   :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>210</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 22px;
font-family: &quot;Rancho&quot;;
color: black;</string>
     </property>
     <property name="text">
      <string>Currency   :</string>
     </property>
    </widget>
    <widget class="QComboBox" name="expensesCategoryComboBox">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>100</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
    font-family: &quot;Rancho&quot;;
    font-size: 16px;
    color: black;
    background: white;
    padding: 5px;
    border: 1px solid #d0d0d0;
}</string>
     </property>
     <item>
      <property name="text">
       <string>Groceries</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Rent</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Utilities</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Transportation</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Entertainment</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Other</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="incomeAmountLineEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">   font-size:16px;
    background-color: white;  
    color: black;            
    border: 1px solid #CCCCCC;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="expensesAmountLineEdit">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>150</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
   font-size:18px;
    background-color: white;  
    color: black;            
    border: 1px solid #CCCCCC;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="currencyComboBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>210</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
    font-family: &quot;Rancho&quot;;
    font-size: 16px;
    color: black;
    background: white;
    padding: 5px;
    border: 1px solid #d0d0d0;
}

</string>
     </property>
     <item>
      <property name="text">
       <string>NPR</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="currencyComboBox_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>200</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
    font-family: &quot;Rancho&quot;;
    font-size: 16px;
    color: black;
    background: white;
    padding: 5px;
    border: 1px solid #d0d0d0;
}

</string>
     </property>
     <item>
      <property name="text">
       <string>NPR</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>221</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Roboto;
font-size: 30px;
color:black;</string>
     </property>
     <property name="text">
      <string>Classification</string>
     </property>
    </widget>
    <widget class="QComboBox" name="classificationComboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>380</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
    font-family: &quot;Rancho&quot;;
    font-size: 18px;
    color: black;
    background: white;
    padding: 5px;
    border: 1px solid #d0d0d0;
}</string>
     </property>
     <item>
      <property name="text">
       <string>Saving goal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Income</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Expenses</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Transfer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Other</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>320</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Roboto;
font-size: 30px;
color:black;</string>
     </property>
     <property name="text">
      <string>Identifiers</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>450</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Roboto;
font-size: 30px;
color:black;</string>
     </property>
     <property name="text">
      <string>Timestamp</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>510</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">    background-color: white;  
    font-size: 16px;
    font-family: &quot;Rancho&quot;;
    color: black;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="identifiersLineEdit">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>380</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">   font-size:18px;
    background-color: white;  
    color: black;            
    border: 1px solid #CCCCCC;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
    <zorder>label_6</zorder>
    <zorder>incomeSouceComboBox</zorder>
    <zorder>label_7</zorder>
    <zorder>label_8</zorder>
    <zorder>label_9</zorder>
    <zorder>expensesCategoryComboBox</zorder>
    <zorder>incomeAmountLineEdit</zorder>
    <zorder>expensesAmountLineEdit</zorder>
    <zorder>currencyComboBox</zorder>
    <zorder>label_10</zorder>
    <zorder>classificationComboBox</zorder>
    <zorder>label_11</zorder>
    <zorder>label_12</zorder>
    <zorder>dateTimeEdit</zorder>
    <zorder>identifiersLineEdit</zorder>
    <zorder>currencyComboBox_2</zorder>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>470</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 26px;
font-family: &quot;Rancho&quot;;
color: red;
   </string>
    </property>
    <property name="text">
     <string>ID/Reference No:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    color: red;
    font-weight: bold;
    font-size: 40px;
</string>
    </property>
    <property name="text">
     <string>FINANCE RECORD</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>60</y>
      <width>841</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/record.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="addRecordButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>690</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">   font-size:30px;
    background-color: white;  
    color: green;            
    border: 1px solid #CCCCCC;
    font-family: &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>Add record</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
